spec/data_migrate/schema_migration_spec.rb

```diff
@@ -65,6 +72,8 @@
           { env_name: Rails.env, spec_name: specification_name }
         elsif Gem::Dependency.new("railties", "~> 7.0").match?("railties", Gem.loaded_specs["railties"].version)
           { env_name: Rails.env, name: specification_name }
+        else
+          { env_name: Rails.env, name: specification_name }
```

Comment:
Does this match the above else if? Should we just remove the previous one and collapse it into this else?